assessment.js

```diff
@@ -1,8 +1,8 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var answer = x;
 
-while (answer !== '' && answer !== 'SeCrEt') {
+while (answer == (x>1 && x<10) {
   answer = ask("Guess my secret? ");
 }
```

Comment:
This answer also has issues. The expression `(x > 1 && x < 10)` will evaluate to true when x is between 1 and 10, but you never actually change the value of x; as a result, the loop runs forever. Since you're getting new `answer` values each time you prompt the user, why not just write  `while (answer > 1 && answer < 10)`.

You're also doing a loose equality test (`==`), which you should avoid. And you actually never set the value of x before you used it in line 4 - this would give you an error if you were to run it.